crates/astria-sequencer/src/genesis.rs

```diff
@@ -18,9 +24,54 @@ pub(crate) struct GenesisState {
 }
 
 impl GenesisState {
-    pub(crate) fn from_path<P: AsRef<Path>>(path: P) -> anyhow::Result<Self> {
-        let file = File::open(path).context("failed to open file with genesis state")?;
-        serde_json::from_reader(&file).context("failed deserializing genesis state from file")
+    /// Load Account information from a CSV file
+    pub(crate) fn from_csv<P: AsRef<Path>>(path: P) -> anyhow::Result<Self> {
+        let mut reader = Reader::from_path(path)?;
+        let mut records: Vec<Account> = Vec::new();
+
+        for result in reader.deserialize() {
+            let record: Account = result?;
+            records.push(record);
+        }
+
+        Ok(Self {
+            accounts: records,
+        })
+    }
+
+    /// Add the Account data from `GenesisState` to the `CometBFT` genesis.json file
+    pub(crate) fn propagate_accounts_to(&self, path: &str) -> anyhow::Result<()> {
+        // build the absolute path to the json file you want to add the accounts to
+        let mut home_path = env::var("HOME")?;
```

Comment:
Please don't read any extra env vars from here. The main entry point should be `config.rs`. `Config` should be the final source of truth for all environmental access and configuration.